docs/develop/02_chain/02_pallets/01_did.md

```diff
@@ -22,15 +22,15 @@ A full DID can then be used to perform certain on-chain actions which include:
 
 A full DID is needed if the user wants to become an Attester or wants to setup delegations.
 A full DID also allows the user to embed a list of URLs, known as service endpoints, into the DID document so that they can be retrieved from the chain as part of the DID document.
-To create a full DID the user first has to create a set of keys and service endpoints:
+To create a full DID the user first has to create a set of keys, where some of them are optional and can be set later on:
```

Comment:
What about 
```suggestion
To create a full DID the user first has to create some keys, and optionally some services
```

We should move away from calling services service endpoints, and just use services 😄.